esp/platform/application_config.cpp

```diff
@@ -89,6 +89,21 @@ static void copyDirectories(IPropertyTree *target, IPropertyTree *src)
     }
 }
 
+void addAuthDomains(const char *authtype, IPropertyTree *legacyEsp)
```

Comment:
picky: authType would be the normal casing.

Reply:
Will fix